app/models/book.rb

```diff
@@ -1,2 +1,8 @@
 class Book < ApplicationRecord
+  validates :genre, presence: true
```

Comment:
If you are applying the same validation to multiple attributes, you can group all of them
```ruby
validates : genre, : author, : image, presence: true
```